dev-packages/e2e-tests/test-applications/node-fastify-app/tests/transactions.test.ts

```diff
@@ -56,6 +56,23 @@ test('Sends an API route transaction', async ({ baseURL }) => {
   expect(transactionEvent).toEqual(
     expect.objectContaining({
       spans: [
+        {
```

Comment:
The additional span here is a consequence of us adding another handler. I think this is fine unless someone has concerns.